style3.css

```diff
@@ -0,0 +1,103 @@
+.openMenu {
+  font-size: 1.6rem;
+  color: #6070ff;
+  cursor: pointer;
+}
+
+.closeMenu {
+  display: block;
+  position: absolute;
+  top: 0;
+  right: 20px;
+  font-size: 2rem;
+  cursor: pointer;
+}
+
+.menuLists {
+  list-style: none;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  width: 100%;
+  flex-direction: column;
+  justify-content: center;
+  gap: 3rem;
+  align-items: center;
+  background: #5e5b5b;
+  transition: top 0.5s ease;
+  display: none;
+}
```

Comment:
- [ ]  Hey, please, let's kindly ensure we follow the design correctly by updating your design to meet this missing part of the design below:

- Use the correct background color and add the missing background bur effects.
- Use the correct font weight for the cancel button.
- Align the links correctly to match the design. 


*Figma design*
![Screenshot from 2023-04-17 17-55-00](https://user-images.githubusercontent.com/59397159/232558021-819a2c16-647e-467f-9ca1-8044d6f340e1.png)

*your design*
![Screenshot from 2023-04-17 17-54-24](https://user-images.githubusercontent.com/59397159/232558032-6faf7e09-619d-4aae-b8f1-cb132ff575fa.png)